source/features/ci-link.css

```diff
@@ -1,7 +1,7 @@
 .rgh-ci-link {
 	position: relative;
 	top: -0.1em;
-	animation: fade-in 0.2s;
+	animation: fade-in 0.2s; /* stylelint-disable-line no-unknown-animations */
```

Comment:
Any way to have stylelint understand this is in a different file?

Reply:
![image](https://user-images.githubusercontent.com/37769974/58323911-0008dc00-7e43-11e9-9ec3-41025df86523.png)

It looks like `stylelint` cannot access references from another file and also there's no way to specify that one exits.

But it also looks like `stylelint` ignores values if placed inside a CSS custom property like:

``` css
.follower-badge {
	--animation-name: fade-in;
	animation: var(--animation-name) 0.2s; /* No errors */
}
```

What do you think about this?

Reply:
I think it’s easier if we disable the rule altogether since we could technically also use GitHub’s animations

Reply:
I've removed the rule from `stylelint-config-xo` now. Just upgrade.

Also see https://github.com/stylelint/stylelint/issues/4088